index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>ITMinds</title>
+  </head>
+  <body>
+    <!---------  Header   ---------->
+    <header>
+      <nav>
+        <div class="top-nav">
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </div>
+        <i class="fas fa-bars fa-2xl menu"></i>
+        <i class="fa-solid fa-x fa-2xl close-menu"></i>
+        <div class="navbar">
+          <a class="logo" href="index.html"
+            ><i class="fa-solid fa-microchip fa-2xl"></i> ITMinds</a
+          >
+          <ul class="nav-links">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#courses">Courses</a></li>
+            <li><a href="#instructors">Instructors</a></li>
+            <li><a href="#partners">Partners</a></li>
+            <li><a href="#">Batches</a></li>
+            <li>
+              <button class="subscribe-btn" type="button">Subscribe</button>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!---------  Main Section   ---------->
+
+    <section class="main">
+      <h4>Empower - Transform - Excel</h4>
+      <h1>UNLOCKING POTENTIAL with IT EDUCATION</h1>
+      <p class="event-details">
+        Join ITMinds' Tech Extravaganza - Unleashing Innovation & Insights!
+        Explore cutting-edge technologies, network with experts, and level up
+        your IT prowess. Don't miss this inspiring seminar!
+      </p>
+      <h2>2023.10.27(FRI) ~ 29(SUN)</h2>
+      <p>@ Expo Center Lahore</p>
+    </section>
+
+    <!---------  Feature Course Section   ---------->
+
+    <section id="courses" class="feature-courses">
+      <div class="feature-cour-heading">
+        <h2>Feature Courses</h2>
+        <hr />
+      </div>
+      <div class="course-container">
+        <div class="course">
+          <img src="images/program_icon_01.png" />
```

Comment:
- [ ] I would like to recommend you to always include `alt` attribute inside you `img`  tag with appropriate text to help vision impaired readers. Kindly check other sections where the `img` tag is used and include the `alt` attribute as needed.